src/components/NavBar/AboutDropdown.tsx

```diff
@@ -1,53 +1,50 @@
-import Link from 'next/link';
-import React, { FC, ReactElement } from 'react';
+import React, { ReactElement } from 'react';
 import { DropdownList, ItemType } from '../Dropdown';
 
-interface IItemLinkProps {
-  href: string;
-}
-const ItemLink: FC<IItemLinkProps> = (props): ReactElement => {
-  const { href, children } = props;
-
-  return (
-    <Link href={href}>
-      <a>
-        <div className="w-full h-full text-gray-700">{children}</div>
-      </a>
-    </Link>
-  );
-};
-
-const items: ItemType[] = [
+const menu = [
   {
-    id: 'about',
-    element: <ItemLink href="/about"> About ADS</ItemLink>,
+    id: 'about-about',
+    path: '/about',
+    label: 'About ADS'
   },
   {
-    id: 'whats new',
-    element: <ItemLink href="/help/whats_new"> What's New</ItemLink>,
+    id: 'about-new',
+    path: '/help/whats_new',
+    label: 'What\'s New'
   },
-  {
-    id: 'blog',
-    element: <ItemLink href="/blog"> ADS Blog</ItemLink>,
+  { 
+    id: 'about-blog',
+    path: '/blog',
+    label: 'ADS Blog'
   },
   {
-    id: 'help',
-    element: <ItemLink href="/help"> ADS Help Pages</ItemLink>,
+    id: 'about-help',
+    path: '/help/',
+    label: 'ADS Help Pages'
   },
   {
-    id: 'legacy',
-    element: <ItemLink href="/help/legacy"> ADS Legacy Services</ItemLink>,
+    id: 'about-legacy',
+    path: '/help/legacy',
+    label: 'ADS Legacy Services'
   },
   {
-    id: 'careers',
-    element: <ItemLink href="/about/careers"> Careers@ADS</ItemLink>,
-  },
-];
+    id: 'about-careers',
+    path: '/about/careers',
+    label: 'Careers@ADS'
+  }
+]
+
+const items: ItemType[] = (() => {
```

Comment:
This one I wasn't sure about, you're trying to map it down to only include `id` and `label`? 
I think you could get rid of the IIFE, since really this is a cheap enough operation to have directly in the function, you can just move it like this:

```js
export const AboutDropdown = (): ReactElement => {
  const handleSelect = (id: string) => {
    window.open(menu.find((item) => id === item.id).path, '_blank', 'noopener,noreferrer');
  };

  // map items to remove unnessary keys
  const items = menu.map(({ id, label }) => ({ id, label }));

  return (
    <DropdownList
      label="About"
...
```

Reply:
If you were concerned with the operation being too expensive or if this component were to render tons of times then I think an optimization would be warranted.

Reply:
I removed `element` for the following reasons:
1) Trying to maintain a consistent look and feel with the dropdown, so only let parent handle the click.
2) Accessibility in the dropdown using keyboard was very tricky

Reply:
The mapping was meant to map them to `ItemType` type.

Reply:
Okay that makes sense, you could also update `ItemType` to remove `element` and add `path` or whatever.
I think its only being used in the navbar right now right?

Reply:
Updated